src/ensureArray.js

```diff
@@ -0,0 +1,23 @@
+import { when, of } from 'ramda';
+import { isNotArray } from './index';
```

Comment:
Insert one empty line between absolute and relative imports pls

Reply:
why eslint did not see that ?

Reply:
May be related to prettier integration. Create a separate issue for this pls.